x-pack/plugins/spaces/server/default_space/create_default_space.ts

```diff
@@ -14,9 +15,10 @@ import { DEFAULT_SPACE_ID } from '../../common/constants';
 interface Deps {
   getSavedObjects: () => Promise<Pick<SavedObjectsServiceStart, 'createInternalRepository'>>;
   logger: Logger;
+  solution?: CloudSetup['onboarding']['defaultSolution'];
```

Comment:
nit: You can use the exposed `OnBoardingDefaultSolution` interface from `@kbn/cloud-plugin/common`